.cursor/README.md

```
@@ -1,111 +1,121 @@
+- **Testing**: `write-unit-tests.md`, `write-api-test.md`, `run-all-tests-and-fix.md`
+- **Git workflows**: `git-commit.md`, `git-push.md`, `create-pr.md`, `fix-git-issues.md`
+- **Documentation**: `add-documentation.md`, `generate-api-docs.md`
+- **Debugging**: `debug-issue.md`, `fix-compile-errors.md`, `docker-logs.md`
+- **Planning**: `roadmap.md`, `setup-new-feature.md`, `clarify-task.md`
+- **Architecture**: `info-architecture.md`, `diagrams.md`, `visualize.md`
+- **Audits**: `security-audit.md`, `security-review.md`, `accessibility-audit.md`, `optimize-performance.md`
 
-- `refine-plan.md`: Checklist for refining implementation plans
-- `debug-plan.md`: Guidelines for investigating failures and creating fix plans
+**Usage pattern:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)